packages/cdk8s/src/names.ts

```diff
@@ -53,16 +53,7 @@ export class Names {
 
     components.push(calcHash(path, HASH_LEN));
```

Comment:
We should use `Node.of(c).addr` instead of manually calculating the hash here.

Reply:
Noted